functions/supplier/debezium-supplier/src/test/java/org/springframework/cloud/fn/supplier/debezium/custom/DebeziumEngineBuilderAutoConfigurationTest.java

```diff
@@ -57,8 +57,8 @@
  */
 @Tag("integration")
 @Testcontainers
-public class DebeziumEngineAutoConfigurationTest {
-	private static final Log logger = LogFactory.getLog(DebeziumEngineAutoConfigurationTest.class);
+public class DebeziumEngineBuilderAutoConfigurationTest {
```

Comment:
Is there any reason that the auto-config test is in a different "custom" subpackage than the auto-config?

Reply:
Yes, because it shows how to implement a `custom` Consumer and `custom` ExecutionService instead of those implemented in the `DebeziumReactiveConsumerConfiguration`.  But i'm open for other package name suggestions.

Reply:
That sounds more like a sample? The standard location for tests is in the same package as the unit it is testing - i would be in favor of sticking w/ that for the tests.